synapse/storage/engines/__init__.py

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from ._base import IncorrectDatabaseSetup
+assert IncorrectDatabaseSetup  # Appease PEP8
```

Comment:
using __all__ might be a cleaner way to appease PEP8. Although you're not the only one to be bitten by this.
https://bugs.launchpad.net/pyflakes/+bug/1178905